Source/WebKit/Platform/IPC/unix/ConnectionUnix.cpp

```diff
@@ -223,11 +220,11 @@ bool Connection::processMessage()
 
     ASSERT(attachments.size() == (messageInfo.isBodyOutOfLine() ? messageInfo.attachmentCount() - 1 : messageInfo.attachmentCount()));
 
-    uint8_t* messageBody = messageData;
+    auto messageBody = messageData;
     if (messageInfo.isBodyOutOfLine())
-        messageBody = oolMessageBody->mutableSpan().data();
+        messageBody = oolMessageBody->mutableSpan();
 
-    auto decoder = Decoder::create({ messageBody, messageInfo.bodySize() }, WTFMove(attachments));
+    auto decoder = Decoder::create({ messageBody.data(), messageInfo.bodySize() }, WTFMove(attachments));
```

Comment:
`messageBody.first(messageInfo.bodySize())`

Creating a span from a pointer and a size is considered unsafe and should be avoided as much as possible.